CHANGELOG.unreleased.md

```diff
@@ -12,6 +12,7 @@
 > Users must be able to say: “I had this issue, happy to know it's fixed”
 
 - [Import/VM/From VMware] Fix `Property description must be an object: undefined` [Forum#61834](https://xcp-ng.org/forum/post/61834) [Forum#61900](https://xcp-ng.org/forum/post/61900)
+- [Import/VM/From VMware] Fix `Cannot read properties of undefined (reading 'stream')` (PR [#6825](https://github.com/vatesfr/xen-orchestra/pull/6825)) [Forum#59879](https://xcp-ng.org/forum/post/59879)
```

Comment:
Link order:

```suggestion
- [Import/VM/From VMware] Fix `Cannot read properties of undefined (reading 'stream')` [Forum#59879](https://xcp-ng.org/forum/post/59879) (PR [#6825](https://github.com/vatesfr/xen-orchestra/pull/6825))
```

Otherwise, good message :+1: